deployment/modules/gcp/gce/tesseract/main.tf

```diff
@@ -86,11 +55,54 @@ resource "google_compute_region_instance_template" "tesseract" {
   }
 
   metadata = {
-    gce-container-declaration = module.gce_container_tesseract.metadata_value
     google-logging-enabled    = "true"
     google-monitoring-enabled = "true"
   }
 
+  metadata_startup_script = <<EOT
+    # A name for the container
+    CONTAINER_NAME="tesseract"
+    # The image it'll run
+    IMAGE="${var.server_docker_image}"
+    
+    # Stop and remove the container if it exists
+    docker stop $CONTAINER_NAME || true
+    docker rm $CONTAINER_NAME || true
+    
+    # Pull the latest version of the container image from Docker Hub
+    docker pull $IMAGE
+
+    # Run docker container from image in docker hub
+    docker run \
+      --name="$CONTAINER_NAME" \
+      --privileged \
```

Comment:
I bet this is required? I had a quick look in the terraform config / instance template / etc, and I couldn't find the equivalent. Is this contained in the docker startup command?